contracts/leverage_trading/src/cancel_take_profit_order.rs

```diff
@@ -135,5 +123,42 @@ impl Contract {
             order_id,
             order_type: OrderType::TakeProfit,
         });
+
+        if let Some(action) = position_action {
+            match action {
```

Comment:
we create match for one option?

Reply:
`position_action: Option<OrderAction>` -> `data_to_close_position: Option<(U128, Order, U128, U128)>`
Removed `match` and `panic!(...)`